selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -1455,13 +1455,18 @@ public void waitSpecifiedValueForLineAndChar(String lineAndChar) {
   }
 
   /** launch the 'Refactor' form by keyboard */
-  public void launchRefactorFormFromEditor() {
+  public void launchLocalRefactor() {
     loader.waitOnClosed();
     Actions action = actionsFactory.createAction(seleniumWebDriver);
     action.keyDown(Keys.SHIFT).sendKeys(Keys.F6).keyUp(Keys.SHIFT).perform();
     loader.waitOnClosed();
   }
 
+  public void launchRefactorForm() {
+    launchLocalRefactor();
```

Comment:
Why do we need to call **launchLocalRefactor** twice?

Reply:
Yes, I have added the comments to the method